csharp/ql/lib/semmle/code/csharp/controlflow/internal/ControlFlowGraphImplShared.qll

```diff
@@ -963,6 +963,20 @@ module Consistency {
     successor = getASuccessor(node, t)
   }
 
+  query predicate simpleAndNormalSuccessors(Node node, SuccessorType t, Node successor) {
+    not node instanceof TEntryNode and
```

Comment:
why exclude `TEntryNode` here?

Reply:
We probably don't need that after all. We need it for `multipleSuccessors` though, since C# allows for methods with multiple bodies (when a method with the same fully qualified name, but different implementations, gets extracted multiple times).

Reply:
ok, makes sense. Although the edges from the entry node should all be labelled "simple", right?

Reply:
Yes, I can strengthen the check to that